tests/migration.go

```diff
@@ -172,3 +173,29 @@ func RunStressTest(vmi *v1.VirtualMachineInstance, vmsize, stressTimeoutSeconds
 		time.Sleep(15 * time.Second)
 	}
 }
+
+func WaitForMigrationRunning(virtClient kubecli.KubevirtClient, vmim *v1.VirtualMachineInstanceMigration, vmi *v1.VirtualMachineInstance, timeout time.Duration) error {
+	return wait.PollImmediate(1*time.Second, timeout, func() (bool, error) {
+		migration, err := virtClient.VirtualMachineInstanceMigration(vmim.Namespace).Get(vmim.Name, &metav1.GetOptions{})
+		if err != nil {
+			return false, fmt.Errorf("failed to get VirtualMachineInstanceMigration %s/%s: %v", migration.Namespace, migration.Name, err)
```

Comment:
This is true for all the other errors composed here: Are you sure the info you add is not already in the original error?
You should probably add at this level that it failed waiting: "failed waiting for migration to run ...".